Formula/msc-generator.rb

```diff
@@ -1,8 +1,9 @@
 class MscGenerator < Formula
   desc "Draws signalling charts from textual description"
   homepage "https://sourceforge.net/p/msc-generator"
-  url "https://downloads.sourceforge.net/project/msc-generator/msc-generator/v7.x/msc-generator-7.2.tar.gz"
-  sha256 "40c7a45e1bce96bc93440b604eb0fd2b7894985fa0a723bf6483565bef026a2e"
+  url "https://gitlab.com/msc-generator/msc-generator/-/package_files/42434730/download"
```

Comment:
Why did the source change? And if it needs to change, why not to https://gitlab.com/msc-generator/msc-generator/-/archive/v7.3.1/msc-generator-v7.3.1.tar.gz ?

Reply:
The project repo has been migrated from Sourceforge to GitLab. The Sourceforge project will be abandoned soon. I updated the "homepage" field now to reflect the change.

I changed the URL to another one which contains the filename and version number. I removed the explicit version number from the formula.